assets/elm/DrinkCard.elm

```diff
@@ -60,4 +60,12 @@ drinkCard index d =
 
 brandNameURL : Drink -> String
 brandNameURL drink =
-    "/brands/" ++ drink.brand |> String.split " " |> String.join "-"
+    let
+        brandName =
+            drink.brand
+                |> String.split "-"
+                |> String.join "_"
```

Comment:
@SimonLab do we _need_ to use an underscore?
Is it _semantic_? Can we just _keep_ hyphens everywhere instead?

Reply:
from #222 
> Spaces in URLs (which appear as %20) should be replaced by dashes
 
Which means that I can't keep hyphens in the the name as spaces are already converted to them. If I keep them I wouldn't know if it's a space or an hyphen when converted back to the real name.

We could use another character other than underscore to convert hyphen?

Reply:
@SimonLab If you are only using the `_` (underscore) as a placeholder and it will not appear in a URL then go for it. But including underscores in URLs is undesirable because _average_ users don't know how to _type_ them on a Mobile keyboard.